openqaoa/workflows/parameters/qaoa_parameters.py

```diff
@@ -176,20 +187,29 @@ def __init__(self,
                                               np.ndarray]] = None,
                  init_hadamard: bool = True,
                  n_shots: int = 100,
-                 seed_simulator: int = None,
                  cvar_alpha: float = 1,
                  noise_model = None,
-                 qubit_layout: Optional[Union[List[int], np.ndarray]] = None):
+                 qubit_layout: Optional[Union[List[int], np.ndarray]] = None,
+                 qiskit_simulation_method: Optional[str] = None,
+                 seed_simulator: Optional[int] = None,
```

Comment:
There are a few extra changes associated with this PR (mostly stuff from qiskit).

Is the seed_simulator backend specific?

Reply:
Yes, it says in the docstring that seed_simulator is for qiskit simulators

Reply:
Any point in renaming it `seed_simulator_qiskit` ?